src/client/java/dev/creesch/WebchatClient.java

```diff
@@ -57,6 +68,55 @@ private void handleMessage(Text message, MinecraftClient client) {
         webInterface.broadcastMessage(jsonChatMessage);
     }
 
+    /**
+     * Checks if the message is a ping.
+     * 
+     * @param message The minecraft text message to process
+     * @param playerName The name of the player
+     * @return True if the message is a ping, false otherwise   
+     */
+    private boolean isPing(Text message, String playerName) {
```

Comment:
Wouldn't it make more sense to determine the player name within the method here?